### PR TITLE
OSDOCS-8455: Adding oc adm upgrade status to Troubleshooting cluster updates

### PR DESCRIPTION
**Pull request by obrown1205**
Version(s):
4.17+

Issue:
[OSDOCS-8455](https://issues.redhat.com/browse/OSDOCS-8455)

Link to docs preview:
https://79593--ocpdocs-pr.netlify.app/openshift-enterprise/latest/updating/troubleshooting_updates/gathering-data-cluster-update.html
https://79593--ocpdocs-pr.netlify.app/openshift-enterprise/latest/updating/updating_a_cluster/updating-cluster-cli.html

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: updating/troubleshooting_updates/gathering-data-cluster-update.adoc [hunk 1]
```diff
@@ -21,6 +21,8 @@ include::modules/gathering-clusterversion-history-console.adoc[leveloffset=+2]
 
 include::modules/gathering-clusterversion-history-cli.adoc[leveloffset=+2]
 
+include::modules/update-upgrading-oc-adm-upgrade-status.adoc[leveloffset=+1]
```
Comment (wking) [inline updating/troubleshooting_updates/gathering-data-cluster-update.adoc:24]: nit: Maybe move this before the `history` sections, but after the (short) log-data/must-gather section?  I'd certainly recommend folks run this command before recommending they check in with cluster history, in terms of what's most likely to help them understand why their update is having trouble.

**Comment by jiajliu** [comment]
cc @evakhoni for a review

**Comment by evakhoni** [comment]
I"m ok with the change

**Review by JoeAldinger — COMMENTED** [review]
/lgtm

**Review by Srivaralakshmi — COMMENTED** [review]
/lgtm

**Comment by Srivaralakshmi** [comment]
> /lgtm

Thanks! FYI, I have set the label to peer-review-done.

**Pull request merged**